exchange/auction_test.go

```diff
@@ -118,6 +118,55 @@ func TestCacheJSON(t *testing.T) {
 	}
 }
 
+func TestIsDebugOverrideEnabled(t *testing.T) {
+
+	type inTest struct {
+		debugHeader string
+		configToken string
+	}
+	type outTest struct {
+		result bool
```

Comment:
IMHO there's no need to define a structure with just a single item. Since this is a test though, my opinion on this is weak. Do whatever you think looks best.

Reply:
yes, modified